codesign/lib/src/file_codesign_visitor.dart

```diff
@@ -93,27 +97,22 @@ Codesign test failed.
 We compared binary files in engine artifacts with those listed in
 entilement.txt and withoutEntitlements.txt, and the binary files do not match.
 *entitlements.txt is the configuartion file encoded in engine artifact zip,
-built by BUID.gn and Ninja, to detail the list of entitlement files.
+built by BUILD.gn and Ninja, to detail the list of entitlement files.
 Either an expected file was not found in *entitlements.txt, or an unexpected
 file was found in entitlements.txt.
 
-This usually happens either during an engine roll.
+This usually happens during an engine roll.
 If this is a valid change, then BUILD.gn needs to be changed.
-Specifically, update [entitlements.txt] or
-[withoutEntitlements.txt] lists, depending on if the file should have macOS
-entitlements applied during codesigning.
+For example, If this is a new binary that need to be code signed with entitlements, add it
```

Comment:
Most people won't know which binaries need entitlements, so I think you should clarify that binaries that will run on a macOS host require entitlements--binaries that run on an iOS device must NOT have entitlements.

Reply:
Yes sir, done.